sklearn/metrics/_plot/tests/test_confusion_matrix_display.py

```diff
@@ -0,0 +1,328 @@
+from numpy.testing import (
```

Comment:
This is the same as the previous test file, with some minor adjustements, right?

Should we also test that both class methods yield the same plots? Maybe checking the `confusion_matrix` attribute is enough

Reply:
Yes, it is the same file but I used a fixture to test both constructor. Since some of the tests already compare to the `confusion_matrix` function output, I don't think that we need to check further.